build/ndm-daemonset/entrypoint.sh

```diff
@@ -2,17 +2,28 @@
 
 export GOTRACEBACK=crash
 
+# openebs base directory inside the container
+OPENEBS_BASE_DIR="/var/openebs"
+# ndm base directory inside the container. It will be the ndm
+# directory inside openebs base directory
+NDM_BASE_DIR="${OPENEBS_BASE_DIR}/ndm"
+
 # set ulimit to 0, if the core dump is not enabled
 if [ -z "$ENABLE_COREDUMP" ]; then
   ulimit -c 0
 else
+  # making sure mountpath inside the container is available
+  if ! [ -d "${NDM_BASE_DIR}" ]; then
+    echo "OpenEBS/NDM Base directory not found"
```

Comment:
nit: I think to represent the message we should use `$OPENEBS_BASE_DIR/$NDM_BASE_DIR`

Reply:
`NDM_BASE_DIR=OPENEBS_BASE_DIR/ndm`. That's why used that error message.